src/postgres/event-handlers/RootPostReationCreated.ts

```diff
@@ -0,0 +1,20 @@
+import { PostId, Post } from '@subsocial/types/substrate/interfaces/subsocial';
+import { insertActivityForPostReaction } from '../insert-activity';
+import { insertNotificationForOwner } from '../notifications';
+import { SubstrateEvent } from '../../substrate/types';
+
+export const onRootPostReactionCreated = async (eventAction: SubstrateEvent, post: Post) => {
+  const { data } = eventAction;
```

Comment:
Fix this copypasta on all post events handlers:

```typescript
const { data } = eventAction;
const follower = data[0].toString();
const postId = data[1] as PostId;
```

it should be something like this everywhere:

```typescript
const { author, postId } = parsePostEvent(eventAction);
```